cdn/dovetail-cdn.yml

```diff
@@ -49,6 +49,27 @@ Resources:
       DistributionConfig:
         Aliases:
           - !Ref DistributionDomain
+        CustomErrorResponses:
+          # unrecognized uri, and s3 object not found
+          - ErrorCachingMinTTL: 300
+            ErrorCode: 403
+            ResponseCode: 404
+            ResponsePagePath: 404.html
+          # recognized uri, arrangement not in redis
```

Comment:
It would be insane if a user got to a `<digest>` that wasn't in redis yet.  So the lambda explicitly returns a 404, and we set a 0-ttl on it, so it gets retried right away.